docs/en/basic/Command-Line-Interface.md

```diff
@@ -997,3 +997,114 @@ For example, `unsetTtl` can be used if a regularly managed file requires manual
 ```console
 $ ./bin/alluxio fs unsetTtl /data/yesterday/data-not-yet-analyzed
 ```
+
+## Table Operations
```

Comment:
Could you add a pointer in the general operations section above? (similar to how we have `fs` detailed)

Reply:
Done